content/ja/handbook/company/working-groups/clickhouse-datastore.md

```
@@ -0,0 +1,111 @@
+---
+
+## 属性
+
+| プロパティ        | 値           |
+|-----------------|-----------------|
+| 作成日           | 2022-12-01 |
+| 終了日           | 2024-04-25 |
+| Slack           | #f_clickhouse（社内からのみアクセス可能）|
+| Google Doc      | [アジェンダ](https://docs.google.com/document/d/1ZZ7fE7s18Yxww9wp0-lO7mFxJmwop3pWvqINCQPNubA/edit#)（社内からのみアクセス可能）|
+| Epic            | [リンク](https://gitlab.com/groups/gitlab-com/-/epics/2070) |
+| 概要 & ステータス | [終了条件の進捗](#exit-criteria-progress)を参照 |
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)